addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ParameterTamperScanRule.java

```diff
@@ -240,4 +235,21 @@ public int getCweId() {
     public int getWascId() {
         return 20;
     }
+
+    private AlertBuilder createAlert(
+            int confidence, String param, String attack, String sb, HttpMessage msg) {
```

Comment:
`sb` → `evidence`

Reply:
If the HttpMessage isn’t being used it doesn’t need to be passed.